packages/react/src/components/info-token/InfoToken.tsx

```diff
@@ -84,10 +85,10 @@ const ModeClassMapping: Record<InfoTokenMode, string> = {
     [InfoTokenMode.Filled]: 'filled',
 };
 
-export const InfoToken: FunctionComponent<InfoTokenProps> = ({mode, size, type, className}) => (
+export const InfoToken: FunctionComponent<InfoTokenProps> = ({mode, size, type, className, svgClassName}) => (
     <Svg
         className={classNames('info-token', ModeClassMapping[mode], SizeClassMapping[size], className)}
         svgName={SvgMapping[type][size]}
-        svgClass={classNames('icon mod-stroke', SizeClassMapping[size], TypeColorMapping[type])}
+        svgClass={classNames('icon mod-stroke', SizeClassMapping[size], TypeColorMapping[type], svgClassName)}
```

Comment:
I dont know if its just me but I feel this component is basically mapping most of the props 1:1.  
I wonder if using the <Svg /> component strait away wouldn't be more simple.

Reply:
I'm not 100% sure what you mean but there are two styling classes being passed down here, one to the span (className) and one to the actual real SVG (svgClass)

Reply:
@dmgauthier Would you like to pass the `Svg` component in the props of the `InfoToken` component?

If so, it goes against the work currently being done to get rid of the `Svg` component and it would have a pretty monstrous impact on the Plasma library but also on the AdminUI project.

I think @lprovost-coveo's solution is the right one and less time consuming